ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -40,17 +39,40 @@ const styles = reactStyles({
   },
   menuLinkSelected: {
     backgroundColor: colorWithWhiteness(colors.primary, 0.95)
+  },
+  libraryTabDivider: {
+    width: '100%',
+    margin: '0.5rem 0',
+    backgroundColor: colorWithWhiteness(colors.dark, .5),
+    border: '0 none',
+    height: 1
   }
 });
 
 const libraryTabEnums = {
```

Comment:
Optional: not your code, but these aren't really enums... maybe libraryTabConfig?